with-sentry/sentry.ts

```diff
@@ -0,0 +1,30 @@
+// Context: https://docs.sentry.io/platforms/javascript/best-practices/shared-environments/
+// This really only applies to content scripts that are executed, but we'll set it up for the whole extension here.
+
+import {
+  BrowserClient,
+  defaultStackParser,
+  getDefaultIntegrations,
+  makeFetchTransport,
+  Scope
+} from "@sentry/browser";
+
+const integrations = getDefaultIntegrations({}).filter((defaultIntegration) => {
+  return !["BrowserApiErrors", "Breadcrumbs", "GlobalHandlers"].includes(
```

Comment:
Nit: better to use a Set and .has, not a blocke4

Reply:
Updated!